### PR TITLE
v0.1.0

### PR DESCRIPTION
**Pull request by DevinDeSilva**
Main Changes.

1. Converted the code into python library.
2. Updated README.md
3. Added space adding logic

### CODE REVIEW

**Review by nipdep — COMMENTED** [review]
LGFM

File: digitgen/digit_generator.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                 allowed_digits=["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"]) -> None:
+        self.digit_size = digit_size
+        self.samples = samples
+        self.image_size = image_size
+        self.allowed_digits = allowed_digits
+
+        self.config = CONFIGURATION
+        self.config["common_configs"]["font_file_loc"] = font_file
+
+        self.memory = {}  # dict.fromkeys(allowed_digits,None)
```
Comment (dilithjay) [inline digitgen/digit_generator.py]: Is the comment here intentional?

File: digitgen/digit_generator.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    def generate(self, space_type="None", num_spaces=None, sectors=None, spaces_per_sector=None):
+        """
+        Generator function of the dataset
+
+        Returns:
+            Sequence[tuple[np.array,dict]]: img_array,annotation
+        """
+        # print(np.unique(random_array,return_counts=True))
```
Comment (dilithjay) [inline digitgen/digit_generator.py]: Another comment here.

**Review by dilithjay — APPROVED** [review]
Tiny thing about some comments. Otherwise, LGTM!

File: digitgen/digit_generator.py (hunk 1 above)
Comment (DevinDeSilva) [reply digitgen/digit_generator.py]: Removed the comment It was added during development

File: digitgen/digit_generator.py (hunk 2 above)
Comment (DevinDeSilva) [reply digitgen/digit_generator.py]: Removed the comment It was added during development

**Pull request merged**